python/cugraph/cugraph/tests/sampling/test_node2vec.py

```diff
@@ -165,8 +163,22 @@ def test_node2vec(
     )
     num_verts = G.number_of_vertices()
     k = random.randint(6, 12)
-    start_vertices = cudf.Series(random.sample(range(num_verts), k), dtype="int32")
+    start_vertices = cudf.Series(
+        random.sample(range(num_verts), k), dtype=start_vertices_type
```

Comment:
Maybe this is a FIXME, but if we're picking samples at random it could make it hard to repro a test failure.  We should somehow make it so tests with specific samples can be re-run in the event that the test fails.

Reply:
Right. The samples are random so that the test doesn't cover a fixed set of start vertices but I do agree it makes it hard to debug. Perhaps we can add a print those start_vertices for debugging purposes? I added a FIXME regarding this